src/irmin-pack/pack.ml

```diff
@@ -191,6 +191,12 @@ struct
         true )
       else false
 
+    let flush ?(all = true) t =
```

Comment:
maybe `index` instead of `all` to make clear that this flag only control the flush of the underlying index.